src/components/BuyTokensDialog.vue

```diff
@@ -33,7 +33,7 @@
             @click="openLink(action.link)"
           >
             <template #prepend>
-              <crypto-icon :crypto="action.icon" />
+              <icon><adamant-icon /></icon>
```

Comment:
Done, but I haven't removed all logic. I created admLink computed property, because, I figured, if this ternary operator was added in the first place, then it needs to stay in one form or another for the link to work properly.

Reply:
That's good